hugo/content/courses/vscode-tricks/ide-tasks.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Tasks
-description: Use tasks to automatic repeitive jobs in the terminal 
+description: Use tasks to automatic repeititive jobs in the terminal 
```

Comment:
Correct word here is repetitive

Reply:
right, I made made the mistake, while fixing the typo 😂. just pushed another fix